client/blocks/eligibility-warnings/index.jsx

```diff
@@ -27,19 +27,34 @@ function getHoldMessages( translate ) {
 			description: '',
 			supportUrl: '',
 		},
+		TRANSFER_ALREADY_EXISTS: {
+			title: translate( 'Installation in progress' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'Installation', { context: 'Navigation item'} )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).